### DISCUSSION TITLE
search doesn't show any results

### DISCUSSION THREAD

**Discussion by toppits** [quodlibet/quodlibet#4584] [Q&A] [answered]
Is there something i need to enable first. No matter in what [browse] view i'm in even listening to the very artist i am searching for, it shows no result

**Comment by declension** [comment, marked as answer]
Hard to say without seeing your queries. Some things to try:

 * Make sure the column you are searching for (e.g. `~artist` or `~composer` is visible) if you're doing a free text search (`foo` rather than `artist=foo`)
 * Check and maybe clear the global filter (as per https://quodlibet.readthedocs.io/en/latest/guide/faq.html#why-don-t-all-my-songs-appear-in-the-song-list-when-searching-for-them)

**Reply by toppits** [reply]
I believe the columns are visible and no global filter
![Screenshot_20240920_113647-kavinsky - nightcall (dustin nguyen remix) ｜ slowed (Insomnish edit) - TADASUKE opus  untitled Ogg Opus  - Quod Libet](https://github.com/user-attachments/assets/ccdeb8f4-4dcd-4bc3-bf71-55dbfe782cd4)
![Screenshot_20240920_113536-kavinsky - nightcall (dustin nguyen remix) ｜ slowed (Insomnish edit) - TADASUKE opus  untitled Ogg Opus  - Quod Libet](https://github.com/user-attachments/assets/6c0d1f58-be78-4f12-b828-627234388b4b)
![Screenshot_20240920_113951-kavinsky - nightcall (dustin nguyen remix) ｜ slowed (Insomnish edit) - TADASUKE opus  untitled Ogg Opus  - Quod Libet](https://github.com/user-attachments/assets/443527cd-335e-41ca-9c3f-9820bbf7c7d7)

**Comment by declension** [comment]
Hard to say but those look like untagged files (or files with unreadable tags). IIRC the ~title gets auto-populated with a synthetic string, which itself isn't searchable (a bit confusing I agree).

You could add `title` and `artist` maybe and see.

**Reply by declension** [reply]
Yes, pretty sure that's it - those are just filenames really and no (relevant) tags are being parsed it seems. 

You could also add the `filename` column to help your searches though
